### PR TITLE
more coverage and change project url to include a actual url

### PR DESCRIPTION
**Pull request by grosser**
@zendesk/samson 

### Risks
- Level: consumers of projects json might fail when they always prefixed the host to the url

### CODE REVIEW

File: app/serializers/project_serializer.rb [hunk 1]
```diff
@@ -2,6 +2,6 @@ class ProjectSerializer < ActiveModel::Serializer
   attributes :id, :name, :url
 
   def url
-    project_path(object)
+    project_url(object)
```
Comment (pschambacher) [inline app/serializers/project_serializer.rb]: Is this actually used? You renamed it but I don't see any change elsewhere?

Comment (grosser) [inline app/serializers/project_serializer.rb]: did not see it being used anywhere ... maybe some angular magic ...

Comment (pschambacher) [inline app/serializers/project_serializer.rb]: 😨

**Comment by pschambacher** [comment]
👍  on green I guess

**Pull request merged**